security-response-policies.md

```diff
@@ -0,0 +1,29 @@
+# Security Response Policy Guidelines
+
+The [Project Proposal Template](project-submission-template.md) specifies that
+a project should state what its security response policy is, if any, when
+being submitted to the eBPF Foundation.
+
+Each Technical Project should have a written security response policy, but the
+details may vary by project.  A written policy is not a prerequisite for
```

Comment:
How much process do we think is reasonable to try to encourage here?

I think that the goal is noble and good, but I also wonder depending on the projects whether this is feasible--depending on how it's governed and the goals of the project, maybe the scope of security impact is relatively low. Would we be OK with a project having a policy that simply says "We accept patches for bugs, but we do not engage with security reports"?

The above is a bit of an extreme example, and maybe we don't need to consider that case until it comes up. But there is a degree of tradeoff here between expectations and imposed governance on submitted projects.